src/main/resources/config/application-localvc.yml

```diff
@@ -12,5 +12,5 @@ artemis:
         url: http://localhost:8000
         build-agent-git-username: buildjob_user # Replace with more secure credentials for production. Required for https access to localvc. This config must be set for build agents and localvc.
         build-agent-git-password: buildjob_password # Replace with more secure credentials for production. Required for https access to localvc. This config must be set for build agents and localvc. You can also use an ssh key
-
+        use-localvc-version-control-access-token: true
```

Comment:
I'd suggest to use the same config for both VCs . You won't use two VCs concurrently.

Reply:
And even more .. it reduces the effort to migrate from one to another because the feature is available for both.

Reply:
that's a good point, I changed it back to make migration easier